packages/vitest/src/types/reporter.ts

```diff
@@ -7,6 +7,8 @@ export interface Reporter {
   onPathsCollected?: (paths?: string[]) => Awaitable<void>
   onCollected?: (files?: File[]) => Awaitable<void>
   onFinished?: (files?: File[], errors?: unknown[]) => Awaitable<void>
+  /** @internal */
+  onFinishedReportCoverage?: () => Awaitable<void>
```

Comment:
Is there any other way? I don't like exposing this even with `@internal` flag 🤔

Reply:
I think it's necessary to notify websocket client in one way or another.

Actually there's one thing possible, which is to expose only `WebSocketEvents.onFinishedReportCoverage` without `Reporter.onFinishedReportCoverage` and it would look like this https://github.com/vitest-dev/vitest/pull/5242/commits/468c057edc305664df0ecc82ed60b32d0e95fab3

Reply:
I like it 👍🏻